### PR TITLE
Fixes #1665 Mutation joinPublicOrganization not complatible with talawa-api

### PR DESCRIPTION
**Pull request by Ayush0Chaudhary**
**What kind of change does this PR introduce?**
It contains a new resolver that is required for mobile app to properly

**Issue Number:**

Fixes https://github.com/PalisadoesFoundation/talawa/issues/1665

**Did you add tests for your changes?**
no

**Snapshots/Videos:**

https://user-images.githubusercontent.com/95746190/226221375-d3e7640e-4663-4347-b7b7-ed51f1f67f5d.mp4

**If relevant, did you update the documentation?**

**Summary**
It contains a resolver that return a list of orgs user have joined

**Does this PR introduce a breaking change?**
no

**Other information**
no

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**
yes

### CODE REVIEW

**Comment by Ayush0Chaudhary** [comment]
@palisadoes

File: src/resolvers/Query/getOrganizationWithUserID.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { QueryResolvers } from "../../types/generatedGraphQLTypes";
+import { Organization } from "../../models";
+
+export const getOrganizationWithUserID: QueryResolvers["getOrganizationWithUserID"] =
+  async (_parent, args) => {
+    return Organization.find({
```
Comment (beingnoble03) [inline src/resolvers/Query/getOrganizationWithUserID.ts:6]: Missing await.

**Comment by Ayush0Chaudhary** [comment]
here is the options the waiting screen provide **logout** and **joinorg**

```
  logout() {
    final user = Hive.box<User>('currentUser');
    final url = Hive.box('url');
    user.clear();
    url.clear();
    navigationService.removeAllAndPush(
      Routes.languageSelectionRoute,
      Routes.splashScreen,
      arguments: '0',
    );
  }

  joinOrg() {
    navigationService.pushScreen(Routes.joinOrg, arguments: '-1');
  }
```

**Comment by Ayush0Chaudhary** [comment]
@xoldyckk

**Comment by xoldyckk** [comment]
@Ayush0Chaudhary Let's not talk in the PR space. Make a new discussions tab:- https://github.com/PalisadoesFoundation/talawa-api/discussions.

**Comment by Ayush0Chaudhary** [comment]
https://github.com/PalisadoesFoundation/talawa/pull/1683

**Comment by Ayush0Chaudhary** [comment]
@xoldyckk, it is returning the complete org list now. 
Understood what you were trying to say🥳

Please merge my 1 line PR. ( Sarcasm intended ) 🫡
@palisadoes @jordan-barrett-jm

**Review by palisadoes — APPROVED** [review]

**Pull request merged**